index.js

```diff
@@ -0,0 +1,231 @@
+const baseCurrencyName = document.querySelector("#base-currency-name");
+const targetCurrencyName = document.querySelector("#target-currency-name");
+const exchangeRate = document.querySelector("#exchange-rate");
+const firstCurrencyAmount = document.querySelector("#first-currency-amount");
+const firstCurrencyList = document.querySelector("#first-currency-list");
+const secondCurrencyList = document.querySelector("#second-currency-list");
+const result = document.querySelector("#result");
+const newRate = document.querySelector("#new-rate");
+const messageContainer = document.querySelector("#message-container");
+messageContainer.style.display = "none";
+const messageText = document.querySelector("#message-text");
+const addError = document.querySelector("#add-error");
+const convertError = document.querySelector("#convert-error");
+const questionContainer = document.querySelector("#question-container");
+questionContainer.style.display = "none";
+const questionText = document.querySelector("#question-text");
+const acceptBtn = document.querySelector("#accept-btn");
+const rejectBtn = document.querySelector("#reject-btn");
+
+const currencyRates = [];
+
+// Adding A New Currency
+
+function setDate() {
+  const timestamp = Date.now();
+  const currentDate = new Date();
```

Comment:
```suggestion
  const currentDate = new Date();
  const timestamp = currentDate.getTime();
```
If you want to be extra sure that they are same dates.